apps/root/src/pages/history/components/historyTable/index.tsx

```diff
@@ -249,16 +271,16 @@ const getTxEventRowData = (txEvent: TransactionEvent, intl: ReturnType<typeof us
     ) : txDate.startOf('day').equals(DateTime.now().minus({ days: 1 }).startOf('day')) ? (
       <FormattedMessage defaultMessage="Yesterday" description="yesterday" />
     ) : (
-      <>{txDate.toLocaleString(DateTime.DATE_SHORT)}</>
+      <>{txDate.toLocaleString({ day: '2-digit', month: '2-digit', year: 'numeric' })}</>
     );
     dateTime = {
       date: formattedDate,
-      time: txDate.toLocaleString(DateTime.TIME_24_WITH_SHORT_OFFSET),
+      time: txDate.toFormat('HH:mm ZZZZ'),
```

Comment:
same with this ones for locale strings since for the locale it also uses the timezone of the user to format the time :D